pkg/proc.attr.hydfab/R/proc_attr_grabber.R

```diff
@@ -980,3 +1075,114 @@ hfab_config_opt <- function(hfab_config,
   return(xtra_cfig_hfab)
 }
 
+######## MISSING COMID-ATTRIBUTES ##########
+fs_attrs_miss_wrap <- function(path_attr_config){
+  #' @title Wrapper searching for comid-attribute data identified as missing
+  #' @details Given missing comid-attribute pairings previously identified
+  #'  from fs_tfrm_attrs.py, and generated as a file by python function
+  #'  `fs_algo.tfrm_attr.write_missing_attrs`
+  #' @param path_attr_config The file path to the attribute config file
+  #' @seealso `fs_algo.tfrm_attr.write_missing_attrs` python
+  #' @export
+
+  # Generate the parameter list
+  Retr_Params <- proc.attr.hydfab::attr_cfig_parse(path_attr_config = path_attr_config)
+
+
+  std_miss_path <- function(dir_db_attrs){
+    #' @title standardize path to file listing all missing attributes
+    #' @param dir_db_attrs The directory to the attribute database
+    #' @seealso `fs_algo.tfrm_attrs.std_miss_path` python package
+    #' @export
+    path_missing_attrs <- file.path(dir_db_attrs,"missing","needed_loc_attrs.csv")
+    return(path_missing_attrs)
+  }
+
+  path_missing_attrs <- std_miss_path(Retr_Params$paths$dir_db_attrs)
+  df_miss <- utils::read.csv(path_missing_attrs,)
```

Comment:
I don't think the comma is needed in utils::read.csv(xxxx,). Not sure if it will matter in terms of functioning.